src/timeline/TimelineEventsVisualizer.jsx

```diff
@@ -275,12 +291,24 @@ class TimelineEventsVisualizer extends Component {
                         lineHeight={40}
                         itemHeightRatio={0.7}
                         itemRenderer={Item}
+                        itemTouchSendsClick={false}
```

Comment:
[JA2]: I also don't think we're using this prop anymore since false is also the default. But if that's wrong, let me know!